Scripts/Interfaces/FunctionalityInjection/IInstantiateUI.cs

```diff
@@ -12,7 +12,7 @@ public interface IInstantiateUI
 
 	public static class IInstantiateUIMethods
 	{
-		internal delegate GameObject InstantiateUIDelegate(GameObject prefab, Transform parent = null, bool worldPositionStays = true);
+		internal delegate GameObject InstantiateUIDelegate(GameObject prefab, Transform parent = null, bool worldPositionStays = true, Transform connectInterfacesOverride = null);
```

Comment:
Just call it rayOrigin.

Another approach (just thinking of this now, sorry) would be to see if the parent transform  is a child of one of the rayorigins by iterating them. I'm not sure if this makes sense in all cases but it would satisfy this one.